MBTI Quiz/dist/script.js

```diff
@@ -205,12 +178,67 @@ function ScrolltoResults(){
   type = firstLetter+secondLetter+thirdLetter+fourthLetter;
   //document.getElementById('resultsmessage').innerHTML = type;
 
+  switch(type) {
+    case "ISTJ":
+      stringType = "is \"The Duty Fulfiller\"";
+      break;
+    case "ISTP":
+      stringType = "is \"The Mechanic\"";
+      break;
+    case "ISFJ":
+      stringType = "is \"The Nurturer\"";
+      break;
+    case "ISFP":
+      stringType = "is \"The Artist\"";
+      break;
+    case "INFJ":
+      stringType = "is \"The Protector\"";
+      break;
+    case "INFP":
+      stringType = "is \"The Idealist\"";
+      break;
+    case "INTJ":
+      stringType = "is \"The Scientist\"";
+      break;
+    case "INTP":
+      stringType = "is \"The Thinker\"";
+      break;
+    case "ESTP":
+      stringType = "is \"The Doer\"";
+      break;
+    case "ESTJ":
+      stringType = "is \"The Guardian\"";
+      break;
+    case "ESFP":
+      stringType = "is \"The Performer\"";
+      break;
+    case "ESFJ":
+      stringType = "is \"The Caregiver\"";
+      break;
+    case "ENFP":
+      stringType = "is \"The Inspirer\"";
+      break;
+    case "ENFJ":
+      stringType = "is \"The Giver\"";
+      break;
+    case "ENTP":
+      stringType = "is \"The Visionary\"";
+      break;
+    case "ENTJ":
+      stringType = "is \"The Artist\"";
+      break;
+    default:
+      stringType = "";
+  }
+
   if (!type.includes('X')){
     document.getElementById(type).style.display='block'
-    document.getElementById('resultsmessage').innerHTML = "Your Result is";
+    document.getElementById('resultsmessage').innerHTML = "Your Result";
+    document.getElementById(type).classList.add("offset-3");
+    document.getElementById(type).parentNode.classList.remove("sibling-fade");
   }
   if(type.includes('X')){
-    document.getElementById('resultsmessage').innerHTML = "Your Result could be one of the below";
+    document.getElementById('resultsmessage').innerHTML = "Your Results";
     var typeResults = ['ISTJ', 'ISTP', 'ISFJ', 'ISFP', 'INFJ', 'INFP','INTJ', 'INTP', 'ESTP', 'ESTJ', 'ESFP', 'ESFJ', 'ENFJ', 'ENTP', 'ENTJ', 'ENFP'];
```

Comment:
You're reusing variables that you're declaring in your switch statement, it would probably make sense to create an array of objects instead with the code and the title associated with it. Can explain this tomorrow :)